src/main/java/com/MTAPizza/Sympoll/pollmanagementservice/dto/vote/VoteCastRequest.java

```diff
@@ -0,0 +1,10 @@
+package com.MTAPizza.Sympoll.pollmanagementservice.dto.vote;
+
+import java.util.UUID;
+
+public record VoteCastRequest(
+        UUID pollId,
```

Comment:
This needs to be removed since the vote service only sends the voting item ID and user id.